2-mandatory/1-weather-report.js

```diff
@@ -13,6 +13,12 @@
 
 function getTemperatureReport(cities) {
     // TODO
+    let temperatureCity = [];
+    for ( let city of cities) {
```

Comment:
I like how you use forOf loop. It's much easier to read from another person's view.